data-archive/archive-jobs/db.properties

```diff
@@ -1,80 +1,92 @@
+# Source databases to be archived
 [Databases]
-DB_NAMES = AUDIT,CREDENTIAL,ESIGNET,IDA,IDREPO,KERNEL,MASTER,PMS,REGPRC,RESIDENT
+DB_NAMES = MASTER
```

Comment:
please add comment on the values of these params.

Reply:
added comments @ckm007